tests/test_interpret.py

```diff
@@ -157,3 +158,73 @@ def test_scan_sequences():
     assert out.end.tolist() == [7, 7, 8, 6]
     assert out.strand.tolist() == ["+", "-", "+", "-"]
     assert out.matched_seq.tolist() == ["CACGTG", "CACGTG", "TGCGTG", "CACGCA"]
+
+
+def test_run_tomtom():
+
+    motifs = {
+        "MA0004.1 Arnt": np.array(
+            [
+                [0.2, 0.95, 0.0, 0.0, 0.0, 0.0],
+                [0.8, 0.0, 1.0, 0.0, 0.0, 0.0],
+                [0.0, 0.05, 0.0, 1.0, 0.0, 1.0],
+                [0.0, 0.0, 0.0, 0.0, 1.0, 0.0],
+            ]
+        ),
+        "MA0006.1 Ahr::Arnt": np.array(
+            [
+                [0.125, 0.0, 0.0, 0.0, 0.0, 0.0],
+                [0.333333, 0.0, 0.958333, 0.0, 0.0, 0.0],
+                [0.083333, 0.958333, 0.0, 0.958333, 0.0, 1.0],
+                [0.458333, 0.041667, 0.041667, 0.041667, 1.0, 0.0],
+            ]
+        ),
+    }
+    df = run_tomtom(motifs=motifs, meme_file=meme_file)
+    assert df.Query_ID.tolist() == [
+        "MA0004.1 Arnt",
+        "MA0004.1 Arnt",
+        "MA0006.1 Ahr::Arnt",
+        "MA0006.1 Ahr::Arnt",
+    ]
+    assert df.Target_ID.tolist() == [
+        "MA0004.1 Arnt",
+        "MA0006.1 Ahr::Arnt",
+        "MA0004.1 Arnt",
+        "MA0006.1 Ahr::Arnt",
+    ]
+    assert np.allclose(df.Optimal_offset, [0.0, 0.0, 0.0, 0.0])
+    assert np.allclose(
+        df["p-value"],
+        [
+            1.339591458093814e-06,
+            0.013090962390804095,
+            0.017318747323072148,
+            3.348979505934935e-07,
+        ],
+        rtol=1e-3,
+    )
+    assert np.allclose(
+        df["E-value"],
+        [
+            2.679182916187628e-06,
```

Comment:
are these number coming from running MEME suite TomTom itself or Jacob's TomTom? If not, would it be possible to check to be sure?

Reply:
Yes, these numbers were checked with both, they're almost identical